app/scripts/controllers/permissions/restricted-methods.test.js

```diff
@@ -69,12 +65,11 @@ describe('restricted methods', function () {
     it('should return accounts in keyring order when none are selected', async function () {
       const keyringAccounts = ['0x7e57e2', '0x7e57e3', '0x7e57e4', '0x7e57e5'];
       const restrictedMethods = getRestrictedMethods({
-        getIdentities: () => {
-          return keyringAccounts.reduce((identities, address) => {
+        getIdentities: () =>
+          keyringAccounts.reduce((identities, address) => {
```

Comment:
Same thing with this. It's not impossible to know where the function begins and ends, of course, because of the indentation, but the braces help in being able to spot it more easily. Also with braces it's possible to jump from the start to the end of a block.